cmd/hailstone/main.go

```diff
@@ -17,26 +18,34 @@ import (
 	hmsl "github.com/davidh-cyberark/brimstone/pkg/hasmysecretleaked"
 	pam "github.com/davidh-cyberark/brimstone/pkg/privilegeaccessmanager"
 	"github.com/davidh-cyberark/brimstone/pkg/utils"
+
+	"github.com/caarlos0/env/v6"
```

Comment:
Please check that this is the latest version.  Looks like v10 might be the latest.